packages/web-components/src/components/cbp-nav-item/cbp-nav-item.scss

```diff
@@ -0,0 +1,68 @@
+/**
+ * @Prop  --cbp-nav-item-button-color: var(--cbp-color-interactive-secondary-darker);
+ * @Prop   --cbp-nav-item-button-color-dark: var(--cbp-color-interactive-secondary-lighter);
+
+ * @Prop   --cbp-nav-item-button-color-hover: var(--cbp-color-interactive-secondary-darker);
+ * @Prop   --cbp-nav-item-button-color-hover-dark: var(--cbp-color-interactive-secondary-lighter);
+
+ * @Prop   --cbp-nav-item-button-color-bg-hover: var(--cbp-color-interactive-secondary-lighter);
+ * @Prop   --cbp-nav-item-button-color-bg-hover-dark: var(--cbp-color-interactive-secondary-darker);
+    
+ * @Prop   --cbp-nav-item-button-border-bottom-hover: var(--cbp-color-interactive-secondary-darker);
+ * @Prop   --cbp-nav-item-button-border-bottom-hover-dark: var(--cbp-color-interactive-secondary-lighter);
+
+ */
+
+ :root {
+    --cbp-nav-item-button-color: var(--cbp-color-interactive-secondary-darker);
```

Comment:
I don't think you need "button" in all of these names. Especially since these are links. You're not styling anything else, so it's really just extra.

Reply:
removed the -button- part from the nav-item cssvars names